backend/content/urls.py

```diff
@@ -2,6 +2,8 @@
 from . import views
 
 urlpatterns = [
+    path('initialize/genre/', views.initialize_genre),
+    path('initialize/contents/', views.initialize_contents),
```

Comment:
@muhwagwa I think making a new url is not a good idea, how about making a new separate file and execute once?